Changelog.md

```diff
@@ -1,6 +1,6 @@
 ## Ignition Fuel Tools 4.x
 
-### Ignition Fuel Tools 4.X.X (20xx-xx-xx)
+### Ignition Fuel Tools 4.2.0 (2020-08-26)
```

Comment:
I think there are many PRs missing on the changelog: https://github.com/ignitionrobotics/ign-fuel-tools/compare/ignition-fuel-tools4_4.1.0...ign-fuel-tools4

Reply:
I think they are all captured in the ign-fuel-tool3  changelog. Should I duplicate all the changelog entries? It seems unnecessary given our policy of migrating changes forward.

Reply:
I guess i could list all of the merge forward pull requests.

Reply:
We've been trying to repeat the entries as they're merged forward or released, so it's clearer when they became available for each distribution. Without that, a Citadel user wouldn't be sure if they get collections from 4.1 or 4.2, for example.

In any case, I don't want to block the PR on this.

Reply:
How's this 75bae8cc43496ccd72e56eea550fe37346ea0de8?